src/game/shard.rs

```diff
@@ -9,27 +9,15 @@ extern "C" {
     #[wasm_bindgen(js_name = "shard")]
     type Shard;
 
+    /// Current shard name.
     #[wasm_bindgen(js_namespace = ["Game"], js_class = "shard", static_method_of = Shard, getter, js_name = name)]
-    fn name() -> JsString;
+    pub fn name() -> JsString;
 
+    /// Shard type. Currently always "normal".
```

Comment:
I know in the docs it says that it's always `"normal"`, but is it worth specifying "in the MMO world..."